druid/single-container-druid/Makefile

```diff
@@ -0,0 +1,18 @@
+DRUID_VERSION?=28.0.0
```

Comment:
Lets update the Makefile following the conventions of AIrflow, Druid, etc..

https://github.com/pubnative/docker-images/pull/200/files#diff-a4396351d4bcc40059cfe8ef85d73ad81c1dd869dfb9a5a3a8570b62073d5499